src/main/resources/resources.properties

```diff
@@ -88,6 +88,7 @@ ui.toolWindow.subTab.exercises.submission.submitExercise=Submit Assignment
 ui.toolWindow.subTab.exercises.submission.selectModule=Please select which module you wish to submit:
 ui.toolWindow.subTab.exercises.submission.selectModuleShort=Select Module
 ui.toolWindow.subTab.exercises.submission.selectAModule=Select a module
+ui.toolWindow.subTab.exercises.submission.tag={0}, {1}, Submission {2}
```

Comment:
Nit: Is this actually a UI string or technical one?

Reply:
it's definitely is **not** a toolWindow `String` :)